changelog/1588834.md

```diff
@@ -0,0 +1,4 @@
+level: minor
+reference: bug 1588834 
+---
+* Allow specifying addtional userdata in AWS Provider worker pools.
```

Comment:
[The changelog doc](https://github.com/taskcluster/taskcluster/blob/master/dev-docs/best-practices/changelog.md) says "Write it in the present tense, as if the change had just been deployed."

```suggestion
AWS Provider worker pools now allow specifying additional userdata beyond that generated by the provider itself.
```